src/dali/abstract_pair_converter_plugin.py

```diff
@@ -87,3 +117,111 @@ def get_conversion_rate(self, timestamp: datetime, from_asset: str, to_asset: st
             )
 
         return result
+
+    def __build_fiat_list(self) -> None:
```

Comment:
Can you move all the new code to ccxt.py? I'd like AbstractPairConverter to change as little as possible, at least in the first iteration. We can refactor code up the inheritance chain later, if needed.

Reply:
Ok, it was my understanding that the fiat conversion code would be in AbstractPairConverter since it is needed for any pair converter plugin that is handling transactions that don't have a stated fiat amount.

Are you saying that all code involved with fiat conversion should go in CCXT and only CCXT?

Reply:
You're right, I now remember we had discussed it and came to that conclusion. It's fine: let's leave it there.